src/components/Gallery.scss

```
@@ -0,0 +1,67 @@
+@use "../styles/_variables.scss" as *;
+
+.Gallery {
```

As above, prefer lowercase class names
```suggestion
.gallery {
```
Remember to change this in the component